### PR TITLE
[doc][trans] submit translation for Yi README: Benchmark + Who can use yi + Misc.

### PR DESCRIPTION
**Pull request by markli404**
This solves #323 

I was using [Yi-34B-Chat](https://huggingface.co/spaces/01-ai/Yi-34B-Chat) to assist me with this translation task, and here is the sample prompts:
image

### CODE REVIEW

**Comment by GloriaLee01** [comment]
@markli404 we would like to invite you to please include the prompt used for translation in your PR description, specifically mentioning that you are using the [Yi-34B-Chat](https://huggingface.co/spaces/01-ai/Yi-34B-Chat) model. This would greatly assist us in better understanding the context and providing the necessary support.🙌

We appreciate your cooperation and participation! 🥳

**Comment by markli404** [comment]
> @markli404 We would like to invite you to please include the prompt used for translation in your PR description, specifically mentioning that you are using the [Yi-34B-Chat](https://huggingface.co/spaces/01-ai/Yi-34B-Chat) model. This would greatly assist us in better understanding the context and providing the necessary support.🙌
> 
> We appreciate your cooperation and participation! 🥳

I just edited my first comment on this PR

File: README_CN.md [hunk 1]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
-- [📊 Base model performance](#-base-model-performance)
+- [📊 Chat模型性能](#-Chat模型性能)
+- [📊 基座模型性能](#-基座模型性能)
 
-### 📊 Chat model performance
-
-Yi-34B-Chat model demonstrates exceptional performance, ranking first among all existing open-source models in the benchmarks including MMLU, CMMLU, BBH, GSM8k, and more.
+### 📊 Chat模型性能
 
+Yi-34B-Chat 模型在包括 MMLU、CMMLU、BBH、GSM8k 等在内的所有现有开源模型的基准测试中表现出色，排名第一。
```
Comment (Cookize) [inline README_CN.md:1003]: Yi-34B-Chat 模型在 MMLU、CMMLU、BBH、GSM8k 等所有开源模型的基准测试中表现出色，排名第一。

File: README_CN.md [hunk 2]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
 
 <details>
-<summary> Evaluation methods and challenges ⬇️ </summary>
+<summary> 评估方法与挑战 ⬇️ </summary>
 
-- **Evaluation methods**: we evaluated various benchmarks using both zero-shot and few-shot methods, except for TruthfulQA.
-- **Zero-shot vs. few-shot**: in chat models, the zero-shot approach is more commonly employed.
-- **Evaluation strategy**: our evaluation strategy involves generating responses while following instructions explicitly or implicitly (such as using few-shot examples). We then isolate relevant answers from the generated text.
-- **Challenges faced**: some models are not well-suited to produce output in the specific format required by instructions in few datasets, which leads to suboptimal results.
+- **评估方式**: 我们使用零样本（zero-shot）和少样本（few-shot）方法对各种基准进行了评估，但 TruthfulQA 除外。
```
Comment (Cookize) [inline README_CN.md:1009]: - **评估方式**: 我们使用零样本（zero-shot）和少样本（few-shot）方法评估了除 TruthfulQA 外的各种基准。

File: README_CN.md [hunk 3]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
+- **零样本与少样本**: 在大部分聊天模型中，更常采用零样本方法。
```
Comment (Cookize) [inline README_CN.md:1010]: - **零样本与少样本**：大部分 Chat 模型常用零样本的方式。

File: README_CN.md [hunk 4]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
 
 <div align="right"> [ <a href="#building-the-next-generation-of-open-source-and-bilingual-llms">Back to top ⬆️ </a> ] </div>
 
-## 📌 Benchmarks 
+## 📌 基准测试 
 
-- [📊 Chat model performance](#-chat-model-performance)
-- [📊 Base model performance](#-base-model-performance)
+- [📊 Chat模型性能](#-Chat模型性能)
```
Comment (Cookize) [inline README_CN.md:998]: - [📊 Chat 模型性能](#-Chat模型性能)

File: README_CN.md [hunk 5]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
+- [📊 基座模型性能](#-基座模型性能)
 
-### 📊 Chat model performance
-
-Yi-34B-Chat model demonstrates exceptional performance, ranking first among all existing open-source models in the benchmarks including MMLU, CMMLU, BBH, GSM8k, and more.
+### 📊 Chat模型性能
```
Comment (Cookize) [inline README_CN.md:1001]: ### 📊 Chat 模型性能

File: README_CN.md [hunk 6]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
-<summary> Evaluation methods and challenges ⬇️ </summary>
+<summary> 评估方法与挑战 ⬇️ </summary>
 
-- **Evaluation methods**: we evaluated various benchmarks using both zero-shot and few-shot methods, except for TruthfulQA.
-- **Zero-shot vs. few-shot**: in chat models, the zero-shot approach is more commonly employed.
-- **Evaluation strategy**: our evaluation strategy involves generating responses while following instructions explicitly or implicitly (such as using few-shot examples). We then isolate relevant answers from the generated text.
-- **Challenges faced**: some models are not well-suited to produce output in the specific format required by instructions in few datasets, which leads to suboptimal results.
+- **评估方式**: 我们使用零样本（zero-shot）和少样本（few-shot）方法对各种基准进行了评估，但 TruthfulQA 除外。
+- **零样本与少样本**: 在大部分聊天模型中，更常采用零样本方法。
+- **评估策略**: 我们的评估策略涉及在明确或隐含地遵循指令（例如使用少量样本示例）的同时生成回应。然后我们从生成的文本中提取相关答案。
```
Comment (Cookize) [inline README_CN.md:1011]: - **评估策略**：我们的评估策略是让模型在明确或隐含地遵循指令（例如使用少量样本示例）的同时生成回应，并从其生成的文本中提取相关答案。

File: README_CN.md [hunk 7]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
+- **面临的挑战**: 一些模型不适合按照少数数据集中的指令所要求的特定格式产生输出，这会导致不理想的结果。
```
Comment (Cookize) [inline README_CN.md:1012]: - **面临的挑战**：一些模型不适用少数数据集中的指令，无法按照所要求的特定格式产生输出。这会导致结果不理想。

File: README_CN.md [hunk 8]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
 
-<strong>*</strong>: C-Eval results are evaluated on the validation datasets
+<strong>*</strong>: C-Eval 是在验证数据集上的结果。
```
Comment (Cookize) [inline README_CN.md:1014]: <strong>*</strong>: C-Eval 的结果来源于验证数据集。

File: README_CN.md [hunk 9]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
 </details>
 
-### 📊 Base model performance
-
-The Yi-34B and Yi-34B-200K models stand out as the top performers among open-source models, especially excelling in MMLU, CMML, common-sense reasoning, reading comprehension, and more.
+### 📊 基础模型性能
 
+Yi-34B 和 Yi-34B-200K 模型作为开源模型中的佼佼者脱颖而出，尤其在 MMLU、CMML、常识推理、阅读理解等方面表现卓越。
```
Comment (Cookize) [inline README_CN.md:1019]: Is CMML supposed to be **CMMLU**?

File: README_CN.md [hunk 10]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
-- **Efforts to Retrieve Unreported Scores**: for scores that were not reported by the original authors (including scores reported with different settings), we try to get results with our pipeline.
-- **Extensive Model Evaluation**: to evaluate the model’s capability extensively, we adopted the methodology outlined in Llama2. Specifically, we included PIQA, SIQA, HellaSwag, WinoGrande, ARC, OBQA, and CSQA to assess common sense reasoning. SquAD, QuAC, and BoolQ were incorporated to evaluate reading comprehension.
-- **Special Configurations**: CSQA was exclusively tested using a 7-shot setup, while all other tests were conducted with a 0-shot configuration. Additionally, we introduced GSM8K (8-shot@1), MATH (4-shot@1), HumanEval (0-shot@1), and MBPP (3-shot@1) under the category "Math & Code".
-- **Falcon-180B Caveat**: Falcon-180B was not tested on QuAC and OBQA due to technical constraints. Its performance score is an average from other tasks, and considering the generally lower scores of these two tasks, Falcon-180B's capabilities are likely not underestimated.
+<summary> 评估方法 ⬇️</summary>
+
+- **结果差异**: 在基准测试开源模型时，我们的流程与公共来源（如 OpenCompass）报告的结果之间存在差异。
```
Comment (Cookize) [inline README_CN.md:1025]: - **结果差异**：在测试开源模型时，我们的流程与公共来源（如 OpenCompass）报告的结果之间存在差异。

File: README_CN.md [hunk 11]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
+- **调查发现**: 深入调查显示，各种模型在提示语、后处理策略和采样技术上的变化可能导致显著的结果差异。
```
Comment (Cookize) [inline README_CN.md:1026]: - **调查发现**: 深入调查显示，在提示语、后处理策略和采样技术的变化可能导致各种模型的结果产生显著差异。

File: README_CN.md [hunk 12]
```diff
@@ -993,62 +993,59 @@ If you're seeking to explore the diverse capabilities within Yi's thriving famil
...
+- **统一的基准测试过程**: 我们的方法论与原始基准一致——使用一致的提示语和后处理策略，并在评估时应用贪心解码（greedy decoding），不对生成内容进行任何后处理。
```
Comment (Cookize) [inline README_CN.md:1027]: - **统一的基准测试过程**：我们的方法论与原始基准一致，即在评估时使用相同的提示语和后处理策略，应用贪心解码生成（greedy decoding），且不对生成内容进行任何修改。

**Review by GloriaLee01 — APPROVED** [review]
🥳

**Pull request merged**